src/components/Search/SearchResultsListItem.tsx

```diff
@@ -82,14 +87,26 @@ function SearchResultsListItem({
           disablePadding
         >
           <ListItemIcon className='search-revision-item-icon search-revision'>
-            <Checkbox
-              className='search-revision-item-checkbox'
-              edge='start'
-              tabIndex={-1}
-              disableRipple
-              data-testid={`checkbox-${index}`}
-              checked={isChecked}
-            />
+            {searchType === 'base' ? (
```

Comment:
noooo please don't add another searchType check :-)
By doing this, we leak the "outside context" to the inside component.
Ideally we should have this type of check nowhere in our app.

Instead, we can configure the component to make it more explicit and straightforward.

```ts
interface {
  ...
  listItemComponent?: 'checkbox' | 'radio'
}

// later

// Defaults to checkbox
const ListItemComponent = listItemComponent === 'radio' ? Radio : Checkbox;

...

<ListItemComponent
  ...
/>
```

Then the information needs to come from `CompareWithBase` and be propagated all the way down to this component.